stories/theme/core/Gmap.css

```diff
@@ -23,3 +23,8 @@
   padding: 5px;
   white-space: nowrap;
 }
+
+.FreeDraw_ToolTip {
+  position: absolute;
+  z-index: 40;
```

Comment:
Is there a z-index management strategy for React-ui?

Reply:
There isn't and this is something I've suggested adding to other repos like ag.js in the past.  We would want something similar to this approach: https://css-tricks.com/handling-z-index/

Reply:
I agree, we should have something set up, honestly, do get this I moved the z-index up in the browser console until it showed up. But, this is just a story so it won't affect anything ag.js side

Reply:
Yes, I love that idea of a z-index management system!